.github/CODEOWNERS

```diff
@@ -133,7 +133,7 @@ pkg/ottl/                                                               @open-te
 pkg/pdatatest/                                                          @open-telemetry/collector-contrib-approvers @djaglowski @fatsheep9146
 pkg/pdatautil/                                                          @open-telemetry/collector-contrib-approvers @dmitryax
 pkg/resourcetotelemetry/                                                @open-telemetry/collector-contrib-approvers @mx-psi
-pkg/stanza/                                                             @open-telemetry/collector-contrib-approvers @djaglowski
+pkg/stanza/                                                             @open-telemetry/collector-contrib-approvers @djaglowski @pjanotti
```

Comment:
This is a much larger scope than I had in mind. I'm not opposed to it if you get more involved in the package as a whole, but I think ownership of `windowseventlogreceiver` clearly implies ownership of the associated operator.

Reply:
Makes sense, updating...

Reply:
[b3c8cb0](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/27667/commits/b3c8cb0af374942286023df6a82df8aebfd5287d)